### DISCUSSION TITLE
Minimap pannable is missing

### DISCUSSION THREAD

**Discussion by ghiscoding** [bcakmakoglu/vue-flow#417] [Q&A] [answered]
It would be nice to have the Minimap "pannable" (as per React-Flow [docs](https://reactflow.dev/docs/api/plugin-components/minimap/#prop-types)) option available to interact with the canvas. React-Flow also have the "zoomable" but I'm more interested into the "pannable" than anything else. It would be nice to have this in the feature, but it's very low priority, so take your time 😉 

#### Prop Types
[`pannable`](https://reactflow.dev/docs/api/plugin-components/minimap/#pannable)
Description: If set to true you can control the React Flow viewport position by dragging the minimap.
Type:            `boolean`
Default:        `false`

**Comment by bcakmakoglu** [comment, marked as answer]
They are both available.
https://vueflow.dev/changelog/additional-components.html#_1-2-0

**Reply by ghiscoding** [reply]
ah great, it might be a good idea to add it into this [Minimap](https://vueflow.dev/guide/components/minimap.html#usage) usage docs. Thanks again 😉

**Reply by bcakmakoglu** [reply]
I'll add it soon 😅
